rpcs3/Emu/RSX/overlay_controls.h

```diff
@@ -457,7 +457,15 @@ namespace rsx
 					{
 						//Resource was not found in config dir, try and grab from relative path (linux)
 						info = std::make_unique<image_info>(("Icons/ui/" + res).c_str());
-
+#ifndef _WIN32
+						if (info->data == nullptr)
+						{
+							char result[ PATH_MAX ];
+							ssize_t count = readlink( "/proc/self/rpcs3", result, PATH_MAX );
```

Comment:
Is `/proc/self/rpcs3` a thing? o_0 Just looked at `/proc/self` on an Arch VM, the only symlink there is `exe`.

So should be something like:

```cpp
#ifdef __linux__
"/proc/self/exe"
#elif __FreeBSD__
"/proc/curproc/file"
#endif
```

Reply:
derp, thanks. Will the FreeBSD ifdef be the only BSD one I need?

Reply:
Not sure if any others are running RPCS3, but I guess you can make that just `else`.